lib/log.js

```diff
@@ -167,6 +215,9 @@ Lug.prototype.activityEvent = function (data) {
 
   this.logger.info('activityEvent', data)
   this.statsd.write(data)
+  // XXX TODO: need to avoid sending the event twice
+  // if it's both an activityEvent and a flowEvent...
+  //sendToAmplitude(data)
```

Comment:
...it seems like here, if the code was moved up to `lib/metrics/events.js`, it would be easy to avoid the duplicate event problem too.